urbanairship-react-native/js/__tests__/CustomEvent.test.ts

```diff
@@ -1,11 +1,11 @@
 /* Copyright Airship and Contributors */
 
-const UACustomEvent =  require('../UACustomEvent')
+import { CustomEvent } from '../CustomEvent';
 
-describe("UACustomEvent Tests", () => {
+describe("CustomEvent Tests", () => {
```

Comment:
Breaking change right? We will need to note these for a release

Reply:
Yeah, this one and (UA)MessageView are the only renamed classes. The typescript definitions will be different, due to minor interface changes and renaming, but the overall API is the same except for removed deprecations.